chart/values.yaml

```
@@ -976,6 +978,7 @@ webserver:
 
   #   # Flask-WTF flag for CSRF
   #   CSRF_ENABLED = True
+  webserverConfigConfigmapName: ~
```

```suggestion
  webserverConfigConfigMapName: ~
```
We should match the casing used by k8s.